pkg/serverless/tags/tags.go

```diff
@@ -47,6 +51,11 @@ func BuildTagMap(arn string, configTags []string) map[string]string {
 	tags = setIfNotEmpty(tags, versionKey, os.Getenv(versionEnvVar))
 	tags = setIfNotEmpty(tags, serviceKey, os.Getenv(serviceEnvVar))
 
+	cleanedRuntime := strings.Replace(os.Getenv(runtimeVar), "AWS_Lambda_", "", 1)
+
+	tags = setIfNotEmpty(tags, runtimeKey, os.Getenv(cleanedRuntime))
```

Comment:
I think the value you're looking for is in `cleanedRuntime` not in os.Getenv(cleanedRuntime)` ?